project.local.json

```diff
@@ -0,0 +1,106 @@
+{
+  "Change": {
+    "Install": {
+      "webBaseDirectory": "www",
+      "webBaseURLPath": ""
+    },
+    "Storage": {
+      "baseFTP": {
+        "class": "\\Change\\Storage\\Engines\\LocalStorage",
+        "basePath": "App/Storage/FTP"
+      }
+    },
+    "Session": {
+      "SaveHandler": {
+        "type": "SimpleRedis",
+        "options": {
+          "server": {
+            "host": "redis",
+            "port": "6379",
+            "timeout": 5
+          },
+          "ttl": 7200
+        }
+      }
+    },
+    "Mail": {
+      "type": "smtp",
+      "host": "smtp.gmail.com",
+      "port": 465,
+      "auth": true,
+      "connection": "login",
+      "username": "XXXX",
+      "password": "XXXX",
+      "ssl": "ssl",
+      "fakemail": [
+      ]
+    },
+    "Svi": {
+      "fakeSvi": "XXXX"
+    },
+    "Sms": {
+      "fakeSms": "XXXX"
+    },
+    "Application": {
+      "development-mode": true
+    },
+    "Presentation": {
+      "AngularDevMode": true
+    },
+    "Dev": {
+      "Resources": {
+        "UI": true,
+        "UI.common": true,
+        "Theme": true,
+        "Theme.base": true
+      }
+    },
+    "Cache": {
+      "block": false,
+      "page": false,
+      "Adapter": {
+        "redis": {
+          "server": {
+            "host": "redis",
+            "port": "6379"
+          }
+        }
+      }
+    },
+    "Db": {
+      "cryptKey": "XXXX",
```

Comment:
You can remove this key as it will be automatically added when installing the project